UnitTests/GitUITests/DiffMergeTools/DiffMergeToolsProviderTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿////using System;
```

Comment:
What's your plan for this file?

Reply:
Ah thanks! It was very late, and I had this file marked as "viewed", so I missed it.
It needs to go, I wrote it as an initial test harness.